packages/shared/tests/schema/judge/post.test.ts

```diff
@@ -1,28 +1,14 @@
 import { v4 } from 'uuid';
 import { Schema } from '../../../src';
+import { commonSchema } from '../../../src/schema/judge';
 
-describe('judge post schema', () => {
-  it('validates judge correctly', () => {
+describe('validate post schema', () => {
+  it(' checks fo', () => {
+    const mergedSchema = commonSchema.merge(Schema.judge.post);
```

Comment:
What were you trying to do here? This should only be using the judge post schema in this file